tests/publiccloud/transfer_repos.pm

```diff
@@ -33,7 +33,10 @@ sub run {
     } else {
         assert_script_run('du -sh ~/repos');
         my $timeout = 2400;
-        assert_script_run("rsync --timeout=$timeout -uvahP -e ssh ~/repos root@" . $args->{my_instance}->public_ip . ":'/tmp/repos'", timeout => $timeout + 10);
+
+        # Mitigate occasional CSP network problems (especially one CSP is prone to those issues!)
+        # Delay of 2 minutes between the tries to give their network some time to recover after a failure
+        script_retry("rsync --timeout=$timeout -uvahP -e ssh ~/repos root@" . $args->{my_instance}->public_ip . ":'/tmp/repos'", timeout => $timeout + 10, retry => 3, delay => 120);
```

Comment:
```suggestion
        script_retry("rsync --timeout=$timeout -uvahP -e ssh ~/repos 'root@" . $args->{my_instance}->public_ip . ":/tmp/repos'", timeout => $timeout + 10, retry => 3, delay => 120);
```
Not releated to your change but from my perspective the other ticks doesn't make much sence